src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityWrappers.cs

```diff
@@ -53,7 +53,7 @@ private static View GetViewFromRefObject(IntPtr refObjectPtr)
 
             if (view is null)
             {
-                NUILog.Error($"RefObject 0x{refObjectPtr:x} is not a View");
+                throw new ArgumentException($"RefObject 0x{refObjectPtr:x} is not a View", nameof(refObjectPtr));
```

Comment:
I just worryed about some case of View's that NUI didn't created. (For example, `Tizen.NUI.Scene3D.Model` create `Tizen.NUI.Scene3D.ModelNode` internally, but they are automatically created only at dali-side. So NUI (actually, Registry) don't know that View. But `ModelNode` is also kind of View.

Reply:
This shouldn't be a problem. The methods in `ViewAccessibilityWrappers.cs` are entry points for `NUIViewAccessible` methods (a C++ subclass of `ControlAccessible`) and `NUIViewAccessible` is only created for (a subset of) C# `View`s.